packages/shared/src/lib/core/layer-2/actions/buildUnwrapAssetTargetAddress.ts

```diff
@@ -3,7 +3,7 @@ import { Bech32Helper } from '@core/utils'
 import { ILayer2TargetAddressParameter } from '../interfaces'
 
 export function buildUnwrapAssetTargetAddress(address: string): ILayer2TargetAddressParameter {
-    const hrp = getNetwork()?.getMetadata().protocol.bech32Hrp ?? ''
+    const hrp = getNetwork()?.protocol.bech32Hrp ?? ''
```

Comment:
`protocol` is missing from `IStardustNetwork`. Check if we also need it in general on `INetwork`.